### ISSUE TITLE
missing value to go with key

### ISSUE THREAD

**Issue by auxym** [andreas-kupries/critcl#62]
critcl works fine in an ActiveTcl 8.5.18 install I have, however, I'm also trying to use it inside an embedded Tcl8.5.9 interpreter. In that last case, `package require tcl` works fine, but as soon as it hits a `::critcl::cproc` I get this error

```
missing value to go with key
    while executing
"dict exists has $t"
    (procedure "ArgsInprocess" line 45)
    invoked from within
"ArgsInprocess $adefs $aoffset"
    (procedure "::critcl::cproc" line 21)
```

Running on win7 with mingw64, I'm pretty sure I have all required packages installed.

**Comment by auxym** [comment]
This does appear to be a bug: changing `dict exists has $t"` to `dict exists $has $t"` appears to run fine. I'm not sure why it didn't show up on my other install.

**Comment by andreas-kupries** [comment]
Is the other install also an 8.5.9 interpreter?
Regardless, a bit of grepping found this, and a nearby location with the same bug.
To be fixed soon, now.

Also of note, when I do
```dict exists has foo```
in my local 8.5.14, and 8.6.4 installs ... No error is thrown.
Result is just `0`, i.e. `false`.
(Hence my question above about the other Tcl interpreter's version)

**Comment by andreas-kupries** [comment]
Addressed in commit 9602ade36c18f352395f95c5cde36c98267ea671
